resources/qml/WelcomePages/CloudContent.qml

```diff
@@ -135,16 +176,28 @@ Item
                 }
             }
 
-            Cura.SecondaryButton
+            // Create an account link
+            Label
             {
-                id: createAccountButton
+                id: createAccountLabel
+                width: parent.width
                 anchors.horizontalCenter: parent.horizontalCenter
-                text: catalog.i18nc("@button","Create account")
-                onClicked: Qt.openUrlExternally(CuraApplication.ultimakerCloudAccountRootUrl + "/app/create")
+                horizontalAlignment: Text.AlignHCenter
+                text:
+                {
+                    var t1 = catalog.i18nc("@text", "Create a free Ultimaker acount")
```

Comment:
Maybe nitpick, but: Ultimaker Account is a name so Account should be capitalized as well

![image](https://user-images.githubusercontent.com/1134120/112133106-9e932f00-8bcb-11eb-966b-7de030cec01f.png)

Reply:
Thx, and I will let Lorenzo known as well